plugins/modules/server.py

```
@@ -77,7 +77,7 @@
   use_public_network:
     description:
       - Attach a public network interface to the server.
-    default: yes
+    default: false
```

As this parameter does not have a default anymore, this line should be removed. The effective default (given by the API) is even "true", so this is currently misleading. If this parameter is not given, a public interface is created.

The default for the `use_private_network` parameter should also be removed from the documentation.